apps/demo/story-config/SegmentedControl.config.tsx

```diff
@@ -1,11 +1,5 @@
 import {DemoConfiguration} from "@config";
-import {
-  DefaultControl,
-  LargeControl,
-  MultiSelectControl,
-  PersistentControl,
-  SegmentedControlDemo,
-} from "@stories";
+import {DefaultControl, LargeControl, SegmentedControlDemo} from "@stories";
 import {SegmentedControl} from "ferns-ui";
 
 export const SegmentedControlConfiguration: DemoConfiguration = {
```

Comment:
"The controls display the current state and related stated" is a little bit of a confusing sentence to me